dissemin/settings/common.py

```diff
@@ -155,6 +155,7 @@
     'django_countries',
     'leaflet',
     'djgeojson',
+    'raven.contrib.django.raven_compat',
```

Comment:
I don't think this should be enabled in `common.py`, as it would also affect dev instances. Also this module is deprecated according to Sentry's docs:
https://docs.sentry.io/clients/python/integrations/django/
We currently use the newer version described here:
https://docs.sentry.io/platforms/python/django/

Reply:
Should be done in https://github.com/dissemin/dissemin/pull/640/commits/53c58db5af1bb2de4b76d3bf4417beb981f67d11. Just be careful that a small migration should probably be done in prod to put the Sentry config in the right place.